### DISCUSSION TITLE
How to put up the Conscious Food Residency page?

### DISCUSSION THREAD

**Discussion by msantolini** [life-itself/community#412] [General]
So I am a bit newbie with the system, but how does the page of the conscious food residency https://github.com/life-itself/community/blob/main/vault/consious-food-residency.md magically appear at https://lifeitself.org/consious-food-residency ? Right now I can't seem to find a way to see it... 
Sorry about the stupid question :D
Thanks!

**Comment by nathenf** [comment]
Hey @msantolini, 

Firstly, since this is not a task I have moved it to discussion. The residency link is https://lifeitself.org/sympoiesis/conscious-food-workshop-june-2023 

If you have created a new page directly in vault it will not show up on the website, because it is not symlinked (which I cannot personally explain but has something to do with need to be linked to the root folder in some way). 

We are actively working to build a guide to everything #393 and it would be great if you could make a list of tasks you are struggling with for us to include in there. 

I hope that helps!

**Comment by msantolini** [comment]
Thanks, this is much clearer. I will check the guide then!

**Comment by rufuspollock** [comment]
Note, this confusion with symlinking would be resolved by #402 item about simplifying site.
